### ISSUE TITLE
Misplacement of top arrow in Safari

### ISSUE THREAD

**Issue by ylian** [Khan/KaTeX#735]
The following is displayed incorrectly in the latest version of Safari on macOS Sierra, retina MBP.

```
\vec x
```
![image](https://user-images.githubusercontent.com/984895/27251567-e71052c6-5349-11e7-940a-b90284017876.png)

It works fine on Chrome.

**Comment by edemaine** [comment]
Just to make sure, are you using `<!DOCTYPE html>`? Does the same behavior appear on our demo page?

**Comment by kevinbarabash** [comment]
I was able to reproduce this using the demo page in Safari 10.

**Comment by ronkok** [comment]
`\vec` uses a combining character, so it has its own unique CSS with `position: relative` and `left: 0.326em`.  The `buildHTML.js` file contains a TODO line suggesting 
````
Fix this in a better way, like by changing the font
````

I'm not volunteering to change the font, but if you're interested, I could give you a temporary solution using an SVG image copied directly from the `\vec` character.

**Comment by kevinbarabash** [comment]
I've noticed while working on https://github.com/Khan/KaTeX/pull/140 that Safari doesn't like combining characters.  :sad:

**Comment by kevinbarabash** [comment]
MathJax is having the same issue, https://github.com/mathjax/MathJax/issues/1737.  There's a link to a fix at the bottom of that issue.

**Comment by ylian** [comment]
Yes indeed. I actually noticed the issue on MathJax before I tried it on KaTeX. Thanks for looking into the issue.

**Comment by ronkok** [comment]
Yesterday, I  tried inserting a non-breaking space:
````
if (group.value.label === "\\vec") {
    accent.value = "\u00A0" + accent.value;
}
````
... and then adjusting the advance value in CSS class `accent-vec` by 0.25em (because nbsp is 0.25 em wide). I found that the arrow came in too low. I don't think I'll get any farther with it. I've got other issues at the moment.

**Comment by kevinbarabash** [comment]
@ronkok thanks for giving it a try.  I'll play around with it when I have some time.

**Comment by edemaine** [comment]
#802 may now fix this bug.

**Comment by ronkok** [comment]
@ylian I believe that release 0.9.0-beta has fixed this bug. Can you confirm?

**Comment by ylian** [comment]
@ronkok Yes, I can confirm this bug is fixed in release 0.9.0-beta. Thanks!
